acceptance-tests/features/support/config.rb

```diff
@@ -25,3 +25,6 @@
 
 ### Require gems such as httprb for interacting with the APIs
 require 'http'
+
+### Require pry gem for debugging purposes
+require 'pry'
```

Comment:
Did you mean to leave this in? This is a debugging tool, we probably should not be including debugging tools in our software.